spec/models/collection_spec.rb

```diff
@@ -501,10 +501,28 @@
       FakeFS.activate!
       FileUtils.mkdir_p(File.join(Avalon::Configuration.lookup('dropbox.path'), 'african_art'))
       FileUtils.mkdir_p(File.join(Avalon::Configuration.lookup('dropbox.path'), 'african_art_2'))
-      expect(Dir).to receive(:mkdir).with(File.join(Avalon::Configuration.lookup('dropbox.path'), 'african_art_3'))
+      expect(Dir).to receive(:mkdir).with(File.join(Avalon::Configuration.lookup('dropbox.path'), 'african_art_3'), 0775)
       collection.send(:create_dropbox_directory!)
       FakeFS.deactivate!
     end
+    it 'creates the dropbox directory with user/group write permissions' do
+      saved_dropbox_path = Avalon::Configuration.lookup('dropbox.path')
+      Avalon::Configuration['dropbox']['path'] = 'spec/fixtures/dropbox'
+
+      collection.name = 'african art'
+      dropbox_dir = File.join(Avalon::Configuration.lookup('dropbox.path'), 'african_art')
+
+      collection.send(:create_dropbox_directory!)
+      expect(File.exists?(dropbox_dir)).to be_truthy
+
+      fstat = File.stat(dropbox_dir)
+      # This is mildly gross. Convert the file mode to a string (representing octal number),
+      # grab the last 3 chars for the UNIX style mode we expect.
+      expect(fstat.mode.to_s(8)[-3..-1]).to eq("775")
```

Comment:
You could lose the string hack and just compare directly to the expected octal:

```ruby
expect(fstat.mode).to eq 0O40775
```

0O is the prefix for octal literals, and the "extraneous" high bit `4` is actually a part of your expectation -- that `dropbox_dir` _is_ a dir, and has it's directory bit set.